### PR TITLE
Fix changing dates on teacher task plans mini-editor

### PR DESCRIPTION
**Pull request by nathanstitt**
On the full builder the main component will re-render and
ripple down to the times.  However the mini-editor is mobx
powered and doesn't bind to the store

### CODE REVIEW

**Comment by pandafulmanda** [comment]
Initial commit changed date on select, but did not save with changed date.

File: tutor/src/components/task-plan/builder/tasking-date-times.cjsx [hunk 1]
```diff
@@ -27,12 +27,6 @@ TaskingDateTimes = React.createClass
   getDefaultProps: ->
     bsSizes: { sm: 8, md: 9 }
 
-  mixins: [BindStoreMixin]
-  bindStore: TaskingStore
-  bindEvent: ->
-    type = if @props.period then @props.period.id else 'all'
-    "taskings.#{@props.id}.#{type}.changed"
-
   getError: ->
```
Comment (nathanstitt) [inline tutor/src/components/task-plan/builder/tasking-date-times.cjsx]: @pandafulmanda without this does the visible dates update on the mini-editor when you create from a cloned plan?

Comment (pandafulmanda) [reply tutor/src/components/task-plan/builder/tasking-date-times.cjsx]: yes

Comment (pandafulmanda) [reply tutor/src/components/task-plan/builder/tasking-date-times.cjsx]: we should chat about this quickly

**Pull request merged**